PartThree.asm

```diff
@@ -0,0 +1,64 @@
+# COMP3410 Program Template
+# Author: William C. Moyer
+# Assignment: PA[X]
+# Date: Date of submission
+
+# Turn in one .asm file per assignment component
+# Remember to submit it as a pull request to the GitHub repo for the assignment
+
+# fibonacci.asm
+# Input: None
+# Output: Print to terminal N Fibonacci numbers
+
+##########################################################
+# .data segment. Constant and variable definitions go here.
+##########################################################
+.data
+firstValue: .word 1 #value of the odd integer being added
+currentValue: .word 1 # partial sum value being held in memory
+
+#secondValue: .word 1
+
+totalSum: .word 0   # total sum. This is the location that the total sum for each 
+space: .asciiz "\n" #creates a line break
+oddNum: .word 1 # odd number value
+###############################################
+# .text segment. Assembly instructions go here.
+###############################################
+	.text
+	addi $k0 , $k0, 1 #sets k0 to 1 this is a counter 
+	
+start: 		bge $k0, 8, EXIT # branch when greater than or equal to 8 
+		
+		
+		la $t0, currentValue #load current value
+		lw $t0, 0($t0)
+		
+		la $t1, firstValue #first value
+		lw $t1, 0($t1) 
+		
+		addi $t2, $t1, 2 #add 2 to the odd integer (first Value). This signifies the odd value to add currentValue by.
+		
+		add $s0, $t0, $t2 # adds the two integers together for a final answer 
+		
+		sw $s0, currentValue # save the partial sum to currentValue
+		sw $t2, firstValue # save the current odd int to firstValue
+		
+	
+		addi $k0 , $k0, 1 #k0++
```

Comment:
-2 $k0 and $k1 are intended for OS use (k for kernel) and should NOT be used as general-purpose registers even though it works.  Plenty of general purpose registers for you to use.